config/hyde.php

```diff
@@ -358,7 +359,7 @@
         // To get started quickly, you can uncomment the defaults here.
         // See the documentation link above for more information.
         'custom' => [
```

Comment:
Still not super happy with this setting name. Claude suggests `'additional'`, not sure if it's really better.

Reply:
Entire suggestion by Claude:

Here's an improved version of the navigation configuration to make the structure more intuitive and consistent.

Here are the key changes and improvements:

1.  Changed "custom" to "additional": This better describes the purpose of this section, which is to add menu items that aren't automatically generated from pages.
2.  Changed the structure of "additional" items: Instead of using a custom `NavigationItem::create()` method, we're using a more standard array format. This makes the configuration more consistent with other parts of the config file and easier to read/write in YAML.
3.  Renamed "subdirectories" to "subdirectory_display": This more clearly indicates that this setting controls how subdirectories are displayed in the menu.
4.  Reordered the sections slightly for better logical flow.

```php
'navigation' => [
    // Determines the order of menu items. Lower values appear first.
    'order' => [
        'index' => 0,
        'posts' => 10,
        'docs/index' => 100,
    ],

    // Customize labels for menu items
    'labels' => [
        'index' => 'Home',
        'docs/index' => 'Docs',
    ],

    // Pages to exclude from the navigation menu
    'exclude' => [
        '404',
    ],

    // Additional menu items (internal or external)
    'additional' => [
        // ['destination' => 'https://github.com/hydephp/hyde', 'label' => 'GitHub', 'priority' => 200],
    ],

    // How to display pages in subdirectories: 'dropdown', 'flat', or 'hidden'
    'subdirectory_display' => 'hidden',
],
```